CONTRIBUTING.md

```diff
@@ -40,6 +40,16 @@ Configuration is available for cmake, [documented elsewhere](https://cmake.org/d
 
 Mac support is pretty early and while it will compile, it's not usable for debugging yet. Builds happen with cmake the same way as Linux.
 
+### Android
+
+To build the components required to debug an Android target, first gather everything from [Dependencies](#dependencies) below, then run:
+```
+mkdir build-android
+cd build-android
+cmake -DBUILD_ANDROID=On -DANDROID_ABI=armeabi-v7a -DANDROID_NATIVE_API_LEVEL=23 ..
+make
+```
```

Comment:
Yes, awful, but easy to get going if you've got Visual Studio installed.  Compiling on Windows is important, so I'll get mingw going to ensure the steps always work.